Taylor and Francis+NEJM.js

```diff
@@ -139,6 +139,9 @@ function scrape(doc, url) {
 					if (item.title.toUpperCase() == item.title) {
 						item.title = ZU.capitalizeTitle(item.title, true);
 					}
+					//ubtue:adding subtitle
+					let subtitle = ZU.xpathText(doc, '//*[@class="NLM_subtitle"]');
+					if (subtitle.length) item.title += ' ' + subtitle;
```

Comment:
Maybe ' - ' as a separator to address cases where the title does not end with a punctiation mark?

Reply:
In the rush I forgot about the seperator, which was actually present in the translator editor. Regulary we separate title and subtitle with colon.